Library/Formula/peco.rb

```diff
@@ -0,0 +1,37 @@
+HOMEBREW_PECO_VERSION="0.2.11"
+class Peco < Formula
+  homepage "https://github.com/peco/peco"
+  if OS.mac?
+    url "https://github.com/peco/peco/releases/download/v#{HOMEBREW_PECO_VERSION}/peco_darwin_amd64.zip"
+    sha1 "1c92cfa7f2dc6d85f11c4225c1b2afff506b141f"
+  elsif OS.linux?
+    url "https://github.com/peco/peco/releases/download/v#{HOMEBREW_PECO_VERSION}/peco_linux_amd64.tar.gz"
+    sha1 "f83dc6c7334377d368cf0be43220639e3525874f"
+  end
```

Comment:
That's how we do things in Homebrew. We'll still ship a binary package but one we've built ourselves.